### ISSUE TITLE
Crime Reports Sensors

### ISSUE THREAD

**Issue by brianjking** [brianjking/homeassistant-config#67]
https://github.com/happyleavesaoc/my-home-automation/blob/master/homeassistant/sensors/crimereports.yaml

```yaml
- platform: crimereports
  zone: 'neighborhood'
  exclude: ['Community Policing']
```

**Comment by brianjking** [comment]
https://github.com/happyleavesaoc/my-home-automation/issues/3

**Comment by brianjking** [comment]
Waiting on https://github.com/home-assistant/home-assistant/pull/6047 to be merged & published.

**Comment by brianjking** [comment]
Looks like https://github.com/home-assistant/home-assistant/pull/6966 was merged into 0.42! Will re-commit once 0.42 is released.
